codegen-server/python/src/main/kotlin/software/amazon/smithy/rust/codegen/server/python/smithy/PythonCodegenServerPlugin.kt

```diff
@@ -44,8 +43,8 @@ class RustCodegenServerPlugin : SmithyBuildPlugin {
         val codegenDecorator = CombinedCodegenDecorator.fromClasspath(context)
 
         // ServerCodegenVisitor is the main driver of code generation that traverses the model and generates code
-        logger.info("Loaded plugin to generate Rust/Python bindings for the server SSDK")
-        ServerCodegenVisitor(context, codegenDecorator).execute()
+        logger.warning("Loaded plugin to generate Rust/Python bindings for the server SSDK for projection ${context.projectionName}")
```

Comment:
Why is this a warning?

Reply:
This is a leftover because I wanted it to print every time. I will change it to info.